modules/project/README.md

```diff
@@ -797,6 +798,29 @@ includedPermissions:
   - resourcemanager.projects.list
 ```
 
+## Quotas
+
+Project and regional quotas can be managed via the `quotas` variable, but require configuring the quota project used by the Terraform provider. This can be done at the provider level via the `user_project_override` and `billing_project` [provider configurations](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference.html#user_project_override). Note that some module functionality will break with this configuration, so we recommend [definining an alias](https://developer.hashicorp.com/terraform/language/providers/configuration#alias-multiple-provider-configurations) for the Google provider changing module so that the quota resource uses it.
+
+```hcl
+module "project" {
+  source = "./fabric/modules/project"
+  name   = "project"
+  quotas = {
+    cpus-ew8 = {
+      service         = "compute.googleapis.com"
+      quota_id        = "CPUS-per-project-region"
+      contact_email   = "user@example.com"
+      preferred_value = 80
+      dimensions = {
+        region = "europe-west8"
+      }
+    }
+  }
+}
+# tftest modules=1 resources=2
```

Comment:
```suggestion
# tftest modules=1 resources=2 e2e
```
Can we add straight away this as E2E? I'm not sure if the contact email has to come from the org, if yes, then `var.group_email` might be sufficient

Reply:
I would not use e2e as this needs a specific provider configuration we can't use as default.